analysis_options.yaml

```diff
@@ -48,7 +48,6 @@ linter:
     - library_prefixes
     - list_remove_unrelated_type
     - literal_only_boolean_expressions
-    - no_adjacent_strings_in_list
```

Comment:
With the new dartfmt here, I think this is lest interesting – we now have indent to make this more clear